web/src/main/java/org/mskcc/cbio/oncokb/controller/DriveAnnotationParser.java

```diff
@@ -601,73 +601,53 @@ private void parseCancer(Gene gene, Set<Alteration> alterations, JSONObject canc
-//                System.out.println(spaceStrByNestLevel(nestLevel + 1) +
-//                    "Last review on: " + MainUtils.getTimeByDate(lastReview));
-//            }
-            String desc = implicationObj.getString("description");
-            evidence.setDescription(desc);
-            setDocuments(desc, evidence);
-
-            if (excludedCancerTypes != null) {
-                evidence.setExcludedCancerTypes(new HashSet<>(excludedCancerTypes));
-            }
-
-            if (implicationObj.has("relevantCancerTypes")) {
-                evidence.setRelevantCancerTypes(new HashSet<>(getTumorTypes(implicationObj.getJSONArray("relevantCancerTypes"))));
-            } else if (relevantCancerTypes != null) {
-                evidence.setRelevantCancerTypes(new HashSet<>(relevantCancerTypes));
-            }
-
-            evidenceBo.save(evidence);
-        }
+//         if (implicationObj.has("description") && !implicationObj.getString("description").trim().isEmpty()) {
+//             // general description
+//             System.out.println(spaceStrByNestLevel(nestLevel + 1) + "Has General Description.");
+//             Date lastEdit = getLastEdit(implicationObj, "description");
+// //            Date lastReview = getLastReview(implicationObj, "description");
+//             Evidence evidence = new Evidence();
+//             evidence.setEvidenceType(evidenceType);
+//             evidence.setAlterations(alterations);
+//             evidence.setGene(gene);
+//             evidence.setCancerTypes(new HashSet<>(tumorTypes));
+//             evidence.setKnownEffect(knownEffectOfEvidence);
+//             evidence.setUuid(getUUID(implicationObj, "description"));
+//             evidence.setLastEdit(lastEdit);
+// //            evidence.setLastReview(lastReview);
+//             if (lastEdit != null) {
+//                 System.out.println(spaceStrByNestLevel(nestLevel + 1) +
+//                     "Last update on: " + MainUtils.getTimeByDate(lastEdit));
+//             }
+// //            if (lastReview != null) {
+// //                System.out.println(spaceStrByNestLevel(nestLevel + 1) +
+// //                    "Last review on: " + MainUtils.getTimeByDate(lastReview));
+// //            }
+//             String desc = implicationObj.getString("description");
+//             evidence.setDescription(desc);
+//             setDocuments(desc, evidence);
+
+//             if (excludedCancerTypes != null) {
+//                 evidence.setExcludedCancerTypes(new HashSet<>(excludedCancerTypes));
+//             }
+
+//             if (implicationObj.has("relevantCancerTypes")) {
+//                 evidence.setRelevantCancerTypes(new HashSet<>(getTumorTypes(implicationObj.getJSONArray("relevantCancerTypes"))));
+//             } else if (relevantCancerTypes != null) {
+//                 evidence.setRelevantCancerTypes(new HashSet<>(relevantCancerTypes));
+//             }
+
+//             evidenceBo.save(evidence);
+//         }
```

Comment:
Let me know if I should delete the code here since its not used

Reply:
@calvinlu3 please go ahead.